src/Uno.UI/Microsoft/UI/Xaml/Controls/Repeater/_Tracing.cs

```diff
@@ -10,7 +10,7 @@ namespace Microsoft.UI.Xaml.Controls
 {
 	internal static partial class _Tracing
 	{
-		[Conditional("TRACE")]
+		[Conditional("REPEATER_TRACE_ENABLED")]
```

Comment:
Personally I prefix all tracing compiler variable by "TRACE_".For instance here I would use "TRACE_ITEMSREAPETER" (I already have a couple of variable like that in Uno)